lib/local-cluster/lib/Cardano/Wallet/Launch/Cluster/ConfiguredPool.hs

```diff
@@ -542,23 +551,27 @@ configurePool metadataServer recipe = do
                             hardForks
                             logCfg'
 
-                let cfg = CardanoNodeConfig
-                        { nodeDir = toFilePath poolDir
-                        , nodeConfigFile = absFilePathOf nodeConfig
-                        , nodeTopologyFile = absFilePathOf topology
-                        , nodeDatabaseDir = toFilePath
-                            $ poolDir </> relDir "db"
-                        , nodeDlgCertFile = Nothing
-                        , nodeSignKeyFile = Nothing
-                        , nodeOpCertFile = Just $ absFilePathOf opCert
-                        , nodeKesKeyFile = Just $ absFilePathOf kesPrv
-                        , nodeVrfKeyFile = Just $ absFilePathOf vrfPrv
-                        , nodePort = Just (NodePort port)
-                        , nodeLoggingHostname = Just name
-                        , nodeExecutable = Nothing
-                        , nodeOutputFile = absFilePathOf <$> nodeOutput
-                        }
+                let cfg =
+                        CardanoNodeConfig
+                            { nodeDir = toFilePath poolDir
+                            , nodeConfigFile = absFilePathOf nodeConfig
+                            , nodeTopologyFile = absFilePathOf topology
+                            , nodeDatabaseDir =
+                                toFilePath
+                                    $ poolDir </> relDir "db"
+                            , nodeDlgCertFile = Nothing
+                            , nodeSignKeyFile = Nothing
+                            , nodeOpCertFile = Just $ absFilePathOf opCert
+                            , nodeKesKeyFile = Just $ absFilePathOf kesPrv
+                            , nodeVrfKeyFile = Just $ absFilePathOf vrfPrv
+                            , nodePort = Just (NodePort port)
+                            , nodeLoggingHostname = Just name
+                            , nodeExecutable = Nothing
+                            , nodeOutputFile = absFilePathOf <$> nodeOutput
+                            , nodeSocketPathFile = NothingK
+                            }
+
                 withConfig
                     $ withCardanoNodeProcess name cfg
-                    $ \socket -> action $ RunningNode socket genesisData vd
+                    $ \NothingK -> action
```

Comment:
```suggestion
                    $ \_ -> action
```

We can always ignore the socket argument — we don't have to statically ensure that we ignore it.